pylint/checkers/exceptions.py

```diff
@@ -337,6 +356,46 @@ def _check_bad_exception_context(self, node):
         ):
             self.add_message("bad-exception-context", node=node)
 
+    def _check_raise_missing_from(self, node: astroid.Raise) -> None:
+        if node.cause is not None:
```

Comment:
Shouldn't this be treated line 345 ?

Reply:
I don't understand.

Reply:
I made another comment that should clarify :)